src/include/rimage/cavs/cavs_ext_manifest.h

```diff
@@ -0,0 +1,249 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Rander Wang <rander.wang@linux.intel.com>
+ */
+
+#ifndef __RIMAGE_CAVS_EXT_MANIFEST_H__
+#define __RIMAGE_CAVS_EXT_MANIFEST_H__
+
+// Structure of ext manifest :
+//ExtendedManifestHeader
+//  ExtendedModuleConfig[0]
+//      SchedulingCapability[]
+//      PinDescr[]
+//  ExtendedModuleConfig[1]
+//      SchedulingCapability[]
+//      PinDescr[]
+//  ...
+//  ExtendedModuleConfig[N]
+//      SchedulingCapability[]
+//      PinDescr[]
+
+/** ExtendedManifestHeader id $AE1 */
+#define EXTENDED_MANIFEST_MAGIC_HEADER_ID	0x31454124
+
+#define FW_MAX_EXT_MODULE_NUM	32
+
+struct uuid_t {
+ uint32_t d0;
+            uint16_t d1;
+            uint16_t d2;
+            uint8_t d3;
+            uint8_t d4;
+            uint8_t d5;
+            uint8_t d6;
+            uint8_t d7;
+            uint8_t d8;
+            uint8_t d9;
+            uint8_t d10;
+};
```

Comment:
packed ?

Reply:
no, I first added packed attribute but fw_parser can't decode it correctly, so I removed it.

Reply:
hmm, these are ABI so should be packed. Best to do a comparison with the other binaries and codebase.